### PR TITLE
a-tiny-JS-world done

### PR DESCRIPTION
**Pull request by Roka20012**
a-tiny-JS-world

[Demo](https://roka20012.github.io/a-tiny-JS-world) |
[Code base](https://github.com/Roka20012/kottans_frontend/tree/master/practical_task/a-tiny-JS-world)

The code is submitted in its own sub-directory and in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by Oodmincheg** [comment]
Hello! Nice speed with exercises :) Can you do some cleaning in  your codebase? I see '39 files changes' and it's real mess :) You can just left your js file in submission folder, or try to use .gitignore file. Feel free ask any questions here or in student chat.

**Comment by Roka20012** [comment]
Sorry, my mistake read the tutorial again. It says to leave only index.js file.
Thank you for having noticed :)

File: submissions/Roka20012/a-tiny-JS-world/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  gender: 'male',
+  legs: 2,
+  hands: 2,
+  saying: 'Hello Jenny!'
+};
+
+let creatures = [dog, cat, woman, man];
+
+creatures.forEach(el => {
+  print(`Hi! I am ${el.species}; My name is <strong>${el.name}</strong>; My gender is <em>${el.gender}</em>;I have ${el.legs} legs;I have ${el.hands} hands;I want to say ${el.saying};`);
```
[the remark this answers is not in the document]
Comment (Roka20012) [reply submissions/Roka20012/a-tiny-JS-world/index.js]: I've fixed it. Do you mean like that?

File: submissions/Roka20012/a-tiny-JS-world/index.js [hunk 2]
```diff
@@ -42,8 +42,8 @@ const man = {
...
 };
 
-let creatures = [dog, cat, woman, man];
-
-creatures.forEach(el => {
-  print(`Hi! I am ${el.species}; My name is <strong>${el.name}</strong>; My gender is <em>${el.gender}</em>;I have ${el.legs} legs;I have ${el.hands} hands;I want to say ${el.saying};`);
+[dog, cat, woman, man].forEach(el => {
+  print(`Hi! I am ${el.species}; My name is <strong>${el.name}</strong>;
+    My gender is <em>${el.gender}</em>;I have ${el.legs} legs;
+    I have ${el.hands} hands;I want to say ${el.saying};`);
```
Comment (Oodmincheg) [inline submissions/Roka20012/a-tiny-JS-world/index.js]: Actually this nicer, but I've meant just space after last semicolons :D

Comment (Oodmincheg) [reply submissions/Roka20012/a-tiny-JS-world/index.js]: `;I have` `hands;I`

Comment (Roka20012) [reply submissions/Roka20012/a-tiny-JS-world/index.js]: Fix it?)Maybe :)

**Comment by Oodmincheg** [comment]
LGTM

**Comment by Roka20012** [comment]
Thank`s have a nice day))

**Pull request merged**